pkgs/development/ocaml-modules/happy-eyeballs/lwt.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildDunePackage
+, happy-eyeballs, cmdliner, duration, dns-client
+, domain-name, ipaddr, fmt, logs, lwt, mtime
+}:
+
+buildDunePackage {
+  pname = "happy-eyeballs-lwt";
+
+  inherit (happy-eyeballs) src version;
+
+  minimalOCamlVersion = "4.08";
+
+  strictDeps = true;
+
+  buildInputs = [
+    happy-eyeballs
+    cmdliner
+    duration
+    dns-client
+    domain-name
+    ipaddr
+    fmt
+    logs
```

```suggestion
    dns-client
    logs
```

"domain-name duration fmt ipaddr logs" is being propagated by "happy-eyeballs".